### ISSUE TITLE
TypeError: window.influxdb.getDatabaseUser is not a function

### ISSUE THREAD

**Issue by itxx00** [influxdb/influxdb#2951]
influxdb-0.9.0-1.x86_64
when click on the database users name, console output this error message:
```
TypeError: window.influxdb.getDatabaseUser is not a function
    at f.e.getDatabaseUser (http://11.123.8.54:8083/javascripts/all.js:33:26189)
    at f.e.showDatabaseUser (http://11.123.8.54:8083/javascripts/all.js:33:27482)
    at mo.functionCall (http://11.123.8.54:8083/javascripts/all.js:29:10826)
    at ia.(anonymous function).compile.n.on.i (http://11.123.8.54:8083/javascripts/all.js:29:17265)
    at f.$get.f.$eval (http://11.123.8.54:8083/javascripts/all.js:28:2394)
    at f.$get.f.$apply (http://11.123.8.54:8083/javascripts/all.js:28:2620)
    at HTMLAnchorElement.<anonymous> (http://11.123.8.54:8083/javascripts/all.js:29:17317)
    at HTMLAnchorElement.ot.event.dispatch (http://11.123.8.54:8083/javascripts/all.js:25:2037)
    at HTMLAnchorElement.ot.event.add.v.handle (http://11.123.8.54:8083/javascripts/all.js:24:31600)
```

**Comment by itxx00** [comment]
window.influxdb.updateDatabaseUser is also not a function

**Comment by ali-graham** [comment]
See also https://github.com/influxdb/influxdb/issues/1829

**Comment by toddboom** [comment]
No longer an issue once #2746 is merged - should be available with v0.9.1.
